docs/en/ingest-management/fleet/fleet-api-docs.asciidoc

```diff
@@ -156,6 +156,194 @@ integration, go to **Integrations**, select an {agent} integration, and click
 **API reference**.
 ****
 
+Example response (truncated for readability):
```

Comment:
Decided to preserve a bit more of this response because it's pretty quick to scroll through it.